raster_loader/cli/snowflake.py

```diff
@@ -147,20 +170,28 @@ def upload(
 
 @snowflake.command(help="Load and describe a table from Snowflake")
 @click.option("--account", help="The Swnoflake account.", required=True)
-@click.option("--username", help="The username.", required=True)
-@click.option("--password", help="The password.", required=True)
+@click.option("--username", help="The username.", required=False, default=None)
+@click.option("--password", help="The password.", required=False, default=None)
+@click.option("--token", help="An access token to authenticate with.", required=False, default=None)
+@click.option("--role", help="The role to use for the file upload.", default=None)
 @click.option("--database", help="The name of the database.", required=True)
 @click.option("--schema", help="The name of the schema.", required=True)
-@click.option("--table", help="The name of the table.", default=None)
+@click.option("--table", help="The name of the table.", required=True)
```

Comment:
Why not doing like in the BQ case and provide a default table name based on the source file/url name?

Reply:
These are the options for the `describe` command, makes no sense to fallback to a default table name, as we are trying to get the info from a specific one

Reply:
In any case, thanks for pointing this out, I just found out that the `describe` command is also available in BQ, and I didn't implement the `--token` option there

Reply:
Done